browser/widevine/widevine_utils.cc

```diff
@@ -28,7 +28,7 @@
 #include "chrome/browser/ui/tabs/tab_strip_model.h"
 #endif
 
-#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
+#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT) && !defined(OS_LINUX)
```

Comment:
I'm not understanding why `!defined(OS_LINUX)` is needed here. cc @simonhong who was just cleaning up these flags and we don't want to confuse them again

Reply:
Yes, these will be removed. Simon added 3 commits with more fixes to Widevine, I haven't added them to the PR yet - was building them locally.

Reply:
Fixed.